dbms/src/Common/CpuId.h

```diff
@@ -0,0 +1,335 @@
+// Copyright 2022 PingCAP, Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
```

Comment:
Is it similar to `contrib/cpu_features/include/cpuinfo_x86.h`

Reply:
found that we can use `TargetSpecific::XXXChecker::runtimeSupport()`, I will remove the useless code soon.